public/API/API_GetClaims.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Validation\Rule;
+use LegacyApp\Community\Enums\ClaimFilters;
+
+/*
+ *  API_GetClaims - returns information about 1000 max set claims.
+ *    k : claim kind - 1 for completed, 2 for dropped, 3 for expired (default: 1)
+ *
+ *  array
+ *   object     [value]
+ *    string     ID                 unique ID of the claim
+ *    string     User               user who made the claim
+ *    string     GameID             id of the claimed game
+ *    string     GameTitle          title of the claimed game
+ *    string     GameIcon           site-relative path to the game's icon image
+ *    string     ConsoleName        console name of the claimed game
+ *    string     ClaimType          claim type: 0 - primary, 1 - collaboration
+ *    string     SetType            set type claimed: 0 - new set, 1 - revision
+ *    string     Status             claim status: 0 - active, 1 - complete, 2 - dropped
+ *    string     Extension          number of thes the claim has been extended
+ *    string     Special            flag indicating a special type of claim
+ *    string     Created            date the claim was made
+ *    string     DoneTime           date the claim is done
```

Comment:
This documentation is mostly pasted from `API_GetActiveClaims.php`. As a result, _roughly half of these documented types are incorrect._ 

In the case of ID, UserRole, GameID, ConsoleID, ClaimType, SetType, Status, Extension, Special, and MinutesLeft, the type is actually `int`, not `string`.

I am happy to update these new docs, I just want to be sure we're also comfortable updating the ones in `API_GetActiveClaims.php` at the same time.

Reply:
We discussed that [here](https://github.com/RetroAchievements/RAWeb/pull/1351#discussion_r1103632477). The general feeling is that it should be okay.

We should update the comments to reflect the changes though.

Reply:
In https://github.com/RetroAchievements/RAWeb/pull/1435/commits/343f3c8b1241c7e1c2d17e9c517e620168f10885 I have updated the documentation for both endpoints.